### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by yuval-piiano**

### CODE REVIEW

File: ci/README.md [hunk 1]
```diff
@@ -126,6 +126,93 @@ The action inputs are:
...
+  when: manual
+
+
+
+```
+
+
+#### How to Trigger the Manual Job
+
+1. Go to your project's CI/CD > Pipelines page.
```
Comment (nerya10) [inline ci/README.md:208]: Should we tell them that they should build their product before triggering the scan ?

File: ci/README.md [hunk 2]
```diff
@@ -126,6 +126,93 @@ The action inputs are:
...
+  stage: scan
+  # Choose a suitable runner
+  tags:
+    - saas-linux-medium-amd64
+  script:
+    # Install requried processes for scan, jq, git, cutl, netcat are needed for Flows scan
+    # docker.io is neededd to work with dind service in ubuntu image 
+    - apt-get update && apt-get install -y maven jq  openjdk-17-jdk git curl netcat apt-transport-https ca-certificates gnupg gnupg-agent software-properties-common docker.io
+    # Build the project locally to get dependecies to local repository (gradle / mvn - in this case mvn)
+    - cd java/bank/source
```
Comment (nerya10) [inline ci/README.md:168]: cd $SUB_DIR

File: ci/README.md [hunk 3]
```diff
@@ -126,6 +126,93 @@ The action inputs are:
...
+        cd ./.piiano/flows/cli
+        # run the scan
+        ./flows-cli.sh $CI_PROJECT_DIR
+      else
+        echo "CI_JOB_TOKEN is not set."
+      fi
+  only:
+    - main
+  when: manual
+
```
Comment (nerya10) [inline ci/README.md:200]: please remove extra lines

**Review by nerya10 — APPROVED** [review]

File: ci/README.md [hunk 4]
```diff
@@ -126,6 +126,90 @@ The action inputs are:
...
+  stage: scan
+  # Choose a suitable runner
+  tags:
+    - saas-linux-medium-amd64
+  script:
+    # Install requried processes for scan, jq, git, curl, netcat are needed for Flows scan
+    # docker.io is neededd to work with dind service in ubuntu image 
+    - apt-get update && apt-get install -y maven jq  openjdk-17-jdk git curl netcat apt-transport-https ca-certificates gnupg gnupg-agent software-properties-common docker.io
+    # Build the project locally to get dependecies to local repository (gradle / mvn - in this case mvn)
+    - cd $SUB_DIR # only needed when sub dir exists and isn't empty
```
Comment (david-piiano) [inline ci/README.md:168]: perhaps add a check if SUB_DIR is non empty and only then chdir inside?

**Pull request merged**